Cargo.toml

```diff
@@ -272,6 +272,17 @@ wasmi = { opt-level = 3 }
 x25519-dalek = { opt-level = 3 }
 yamux = { opt-level = 3 }
 zeroize = { opt-level = 3 }
+
 [profile.release]
 # Substrate runtime requires unwinding.
 panic = "unwind"
+
+[profile.production]
```

Comment:
Should we not call this `benchmarking`?

Reply:
The name here is the same as Polkadot. Any reason not to have it the same?

Reply:
Yeah I know that it is the same as in Polkadot. But this is here about benchmarking and not "production". What is production for Substrate?

Reply:
I would prefer using the same name even if there is no production for polkadot. Reason is that we might want to have such a profile inside the `wasm-builder`, too. And that we would name `production`. Then we are in the situation where `benchmarking` in substrate should activate `production` in wasm-builder. I mean it is not the end of the world but It would be much easier to just name it the same.

Reply:
Okay, fine by me.